packages/system/src/typography.test.js

```diff
@@ -167,6 +168,24 @@ describe('Web and iOS', () => {
         expect(c).toStrictEqual(expectedNoTheme);
       });
     });
+
+    describe('textAlign', () => {
+      it('Should return values for text align prop', () => {
+        const expectedTextAlign = css({ textAlign: 'center' });
+
+        const useSystemTextAlign = textAlign({ textAlign: 'center' });
+
+        expect(useSystemTextAlign).toStrictEqual(expectedTextAlign);
+      });
+
+      it('Should return values for text align prop when using the abbreviation version', () => {
+        const useSystemTextAlign = css({ textAlign: 'center' });
+
+        const textAlignProperty = textAlign({ ta: 'center' });
+
+        expect(textAlignProperty).toStrictEqual(useSystemTextAlign);
+      });
```

Comment:
I think we could make just one scenario testing both props. It would also assert if `textAlign` === `ta`

```suggestion
            it('Should return values for textAlign prop', () => {
        const expectedTextAlign = css({ textAlign: 'center' });

        const ta1 = textAlign({ textAlign: 'center' });
        const ta2 = textAlign({ ta: 'center' });

        expect(ta1).toStrictEqual(ta2);

        const textAlignOptions = [ta1, ta2];
        textAlignOptions.map(ta => expect(ta).toStrictEqual(expectedTextAlign));
      });
```

What do you think?

Reply:
That's awesome! It gives a better visualization of the test. I'm commiting the suggestion 🚀 !!